scripts/create_android_strings.js

```diff
@@ -196,7 +196,7 @@ function processResult(context, lang, langJson, stringXmlJson) {
     _.forEach(langJsonToProcess, function (val, key) {
         // positional string format is in Mac OS X format. change to android format
         val = val.replace(/\$@/gi, '$s');
-        val = val.replace(/\'/gi, "\\'");
+        val = val.replace(/'/gi, "\\'");
```

Comment:
I hope this doesnt break stuff. probably wont but I dont know regex well enought. 😂

Reply:
My IDE detected it as a "Redundant character escape in RegExp" and indeed quotes are not reserved characters.